.teamcity/src/subprojects/build/core/CoreBuild.kt

```diff
@@ -9,10 +9,7 @@ import subprojects.build.*
 class CoreBuild(private val osJVMComboEntry: OSJDKEntry) : BuildType({
     id("KtorMatrix_${osJVMComboEntry.osEntry.name}${osJVMComboEntry.jdkEntry.name}".toExtId())
     name = "${osJVMComboEntry.jdkEntry.name} on ${osJVMComboEntry.osEntry.name}"
-    artifactRules = """
-                        +:**/build/**/*.jar
-                        $reportArtifacts
-                    """.trimIndent()
+    artifactRules = "+:**/build/**/*.jar\n$reportArtifacts"
```

Comment:
Please remove all code duplication, e.g. new lines characters are duplicated.

Reply:
Sorry, which new line character is duplicated? The \n should be where they are.

Reply:
You have `artifactRules` in 3 places and the gluing strings logic is duplicated as a symbol `\n`